docs/cli/network.rst

```diff
@@ -0,0 +1,93 @@
+.. _ref_cli_edgedb_network:
+
+===================================
+Network Usage by Command-Line Tools
+===================================
+
+Generally command-line tools connect only to the database host. But
+there are two exceptions:
+
+1. At the start of the command-line tool it checks if version is up to
+   date. :ref:`Details <ref_cli_edgedb_version_check>`
+2. The ``edgedb server`` family of commands and ``edgedb self-update``
+   discover package versions and docker images and also invoke package
+   managers and docker engine to do some work.
```

Comment:
```suggestion
   managers and the docker engine to do some work.
```

Reply:
Would be great to explicitly clarify what "some work" is.

Reply:
I'd actually replace "some work" with something like "index updates" hyperlinked to the `ref_cli_edgedb_net_server` ref.